README.md

```diff
@@ -43,3 +29,432 @@ let api = namespace.fromCompactRefract(compactRefract);
+The optional content-type of the element's content.
+
+```js
+// Get the content-type
+console.log(copy.contentType);
+
+// Set the content-type
+copy.contentType = 'text/markdown';
+```
+
+### DataStructure ([ObjectElement](https://github.com/refractproject/minim#objectelement))
+This element describes a data structure.
+
+### Resource ([ArrayElement](https://github.com/refractproject/minim#arrayelement))
+
+#### Properties
+
+##### resource.href
+The URL template of this resource.
+
+```js
+// Get the href
+console.log(`URL: ${resource.href}`);
+
+// Set the href
+resource.href = '/foo/{id}';
+```
+
+##### resource.hrefVariables
+The description of any variables present in the `resource.href` URL template.
+
+```js
+// Get the href variables
+console.log(resource.hrefVariables.keys());
+
+// Set the href variables
+resource.hrefVariables = {
+  id: 'foo'
+}
+```
+
+##### resource.transitions
+Get an array element of all child elements with the element name `transition`. This property is **ready-only**.
+
+```js
+for (const transition of resource.transitions) {
+  console.log(`Transition: ${transition.title}`);
+}
+```
+
+##### resource.dataStructure
+Get the first child element with the element name `dataStructure`. This property is **ready-only**.
+
+```js
+console.log(resource.dataStructure.keys());
+```
+
+### Transition ([ArrayElement](https://github.com/refractproject/minim#arrayelement))
+This element represents a resource transition.
+
+#### Properties
+
+##### transition.method
+Get the HTTP method of the transition, if there is one, by finding the first HTTP request and inspecting its method. This property is **read-only**.
+
+```js
+let method = transition.method;
+```
+
+##### transition.relation
+Defines a relationship to another resource or transition. Useful for hypermedia.
+
+```js
+// Get the relation
+console.log(`Relation: ${transition.relation}`);
+
+// Set the relation
+transition.relation = '...';
+```
+
+##### transition.href
+Overrides the resources URL template with one specific to this transition.
+
+```js
+// Get the herf
+console.log(`URL: ${transition.href}`);
+
+// Set the href
+transition.href = '/foo/{id}';
+```
+
+##### transition.computedHref
+Gets either the transition's `href` or the first transaction's request's `href` if it exists, otherwise returns `null`. This property is **read-only**.
+
+```js
+console.log(`URL: ${transition.computedHref}`);
+```
+
+##### transition.hrefVariables
+The description of any variables present in the `transition.href` URL template.
+
+```js
+// Get the href variables
+console.log(transition.hrefVariables.keys());
+
+// Set the href variables
+transition.hrefVariables = {
+  id: 'foo'
+}
+```
+
+##### transition.data
+The data structure describing the body payload for this transition.
+
+```js
+// Get the data attributes
+console.log(transition.data.toRefract());
+
+// Set the data attributes
+transition.data = minim.toElement({one: 1});
+```
+
+##### transition.contentTypes
```

Comment:
I don't see anything about contentTypes in [Transition Element](https://github.com/refractproject/refract-spec/blob/master/namespaces/api-description-namespace.md#transition-element) of the spec. I wonder where this came from.